src/examples/libpmem2/map_multiple_files/map_multiple_files.c

```diff
@@ -0,0 +1,182 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright 2020, Intel Corporation */
+
+/*
+ * map_multiple_files.c -- implementation of virtual address allocation example
+ */
+
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#ifndef _WIN32
+#include <unistd.h>
+#else
+#include <io.h>
+#endif
+#include <libpmem2.h>
+
+int
+main(int argc, char *argv[])
+{
+	char *addr;
+	int *fd;
+	int ret = 1;
+	struct pmem2_config *cfg;
```

Comment:
We don't adhere to C89 ;)
I'd prefer it if you moved all of these variables to where they are used first. Especially the pointers.